scripts/jirautils.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# This script contains jira related utilties for Jenkins.
+
+
+import click
+from jira import JIRA
+
+
+@click.command()
+@click.option('--tag',
+              help='Jenkins build tag',
+              required=True)
+@click.option('--link',
+              help='Link to related build in Jenkins UI',
+              required=True)
+@click.option('--project',
+              help='Name of target repo',
```

Comment:
The default set for `project` on line 487 of common.groovy is `"UG"` which is not the name of a repo so either that or this description should be modified.